src/Tribe/Date_Utils.php

```diff
@@ -1167,6 +1167,34 @@ public static function unescape_date_format( $date_format ) {
 			// Why so simple? Let's handle other cases as those come up. We have tests in place!
 			return str_replace( '\\\\', '\\', $date_format );
 		}
+
+		/**
+		 * Gets the timezone string associated with an event.
+		 *
+		 * Will fall back to WP timezone string and to system timezone string in this order.
+		 *
+		 * @since TBD
+		 *
+		 * @param $event_id
+		 *
+		 * @return mixed|string|void
+		 */
+		public static function get_event_timezone_string( $event_id ) {
```

Comment:
Got a couple of thoughts on this:

1) Here in Tribe Common we should avoid event-specific stuff - that ought to go in TEC
2) What's the advantage of this method over `Tribe__Events__Timezones::get_event_timezone_string()`? If there's a difference, can it be reconciled and can we just alter the existing method of the same name?

Reply:
closing this pr to use an existing method.